docs/examples/prototypes.md

```diff
@@ -1,45 +1,77 @@
 
+![amadeus-flight-price-analysis-django](../images/amadeus-flight-price-analysis-django.png)
 
+### amadeus-hotel-area-safety-pois-django
+amadeus-hotel-area-safety-pois-django ([Link to GitHub](https://github.com/amadeus4dev/amadeus-hotel-area-safety-pois-django){:target="\_blank"}) built by Django and [Python SDK](../programming/python.md), It demonstrate the safety information, POIs and tours for a chosen hotel on the map, using the following APIs:
+
+* [Hotel Search](https://developers.amadeus.com/self-service/category/hotel/api-doc/hotel-list){:target="\_blank"}: shows hotels on the map
+* [Points of Interest](https://developers.amadeus.com/self-service/category/destination-content/api-doc/points-of-interest){:target="\_blank"}: shows POIs around the hotel
+* [Safe Place](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/safe-place){:target="\_blank"}: shows safety information for the area each hotel is located
+* [Tours and Activities](https://developers.amadeus.com/self-service/category/destination-content/api-doc/tours-and-activities){:target="\_blank"}: shows bookable tours and activities around the hotel
+* [HERE Maps](https://developer.here.com/){:target="\_blank"}: displays a map with markers and text bubbles
+
+![amadeus-hotel-area-safety-pois-django](../images/amadeus-hotel-area-safety-pois-django.png)
+
+### amadeus-travel-restrictions-node
+
+amadeus-travel-restrictions-node ([Link to GitHub](https://github.com/amadeus4dev/amadeus-travel-restrictions-node){:target="\_blank"}) built by Node, Express and [Node SDK](../programming/node.md), It demonstrate Travel Restrictions for USA displayed on a map. We use the following APIs:
+
+* [Travel Restrictions](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/travel-restrictions){:target="\_blank"}: returns Travel Restrictions
+* [HERE Maps for JavaScript](https://developer.here.com/): displays a map with markers and text bubbles
+
+![amadeus-travel-restrictions-node](../images/amadeus-travel-restrictions-node.png)
+
+## Prototypes from community 
+
+We have many other prototypes or demo apps that developers in our community built and shared! Explore them in [Amadeus for Developers -Examples GitHub](https://github.com/amadeus4dev-examples){:target="\_blank"} 
+
+!!! warning
+    Projects from communities are examples that have been built and managed by developer community and it is not supported or maintained by Amadeus for Developers team. The projects may not up-to-date. 
+
+!!! danger
+    The list of community projects is working in progress.
```

Comment:
Hey Minji, why do we need this danger section?

Reply:
the list is not completed anna, missing many prototypes :) i need to work on the whole list and make better table